File: treinos/treino001/app/src/main/java/aluno/gabriel/treino001/view/SplashActivity.java
package aluno.gabriel.treino001.view;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import aluno.gabriel.treino001.R;
import aluno.gabriel.treino001.model.Usuario;

public class SplashActivity extends AppCompatActivity {
    Usuario u1;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        trocarTela();
    }

    protected void trocarTela(){
        boolean Handler = new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                u1 = new Usuario(
                        "Gabriel Viticov Plata de Souza",
                        "gabrielviticvo@gmail.com",
                        "3358btQ!;;786"
                );

                Intent intent = new Intent(SplashActivity.this, MainActivity.class);
                Bundle bundle = new Bundle();
                bundle.putString("name", u1.getNome());
                bundle.putString("email", u1.getEmail());
                bundle.putString("password", u1.getSenha());
                intent.putExtras(bundle);
                startActivity(intent);
                finish();
            }
        },5000);
    }
}

File: treinos/treino008/app/src/main/java/aluno/gabriel/treino008/view/SplashActivity.kt
package aluno.gabriel.treino008.view

import aluno.gabriel.treino008.R
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import android.os.Looper

class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        trocarTela()
    }

    protected fun trocarTela(){
        Handler(Looper.getMainLooper()).postDelayed({
            startActivity(Intent(this@SplashActivity, MainActivity::class.java))
            finish()
        },3000)
    }
}